helm-values.tf

```diff
@@ -30,5 +30,5 @@ data "template_file" "prometheus_kv_mapping" {
 resource "local_file" "helm_values_file" {
   count    = "${length(var.prometheus_labels) != 0 ? 1 : 0}"
   content  = "${data.template_file.helm_values.rendered}"
-  filename = "${path.cwd}/helm-values.yaml"
+  filename = "helm-values.yaml"
```

Comment:
I would keep using the old notation `"${path.cwd}/helm-values.yaml"` as it makes it explicit where the file will end up. If you remove the `${path.cwd}` part, this will be dependent on the Terraform default behavior, which poses two small issues:
- if Terraform changes the default behavior this will stop working
- you have to go and search in the Terraform documentation what's the default behavior, which I haven't been able to find so far 😛

Reply:
the cwd approach does not work. it saves the current pwd in the state, which introduces unnecessary changes if a different person runs it.

The previous solution in this PR `"${substr("${path.module}/helm-values.yaml", length(path.cwd) + 1, -1)}"` also does not work, as that saves the file in the module folder under `.terraform`

I think the plain notation as it is now is the best. it always ends up in the current directory where you run terraform.

Reply:
I've added documentation and a comment in the code to clarify this.

Reply:
ok, I was under the impression this was purely an esthetical change.